### PR TITLE
[Doc]Add 22.10 download page[skip ci]

### PR DESCRIPTION
**Pull request by viadea**
- Add 2210 download page
- fix GCP custom image script
- Modify Doc for 2210: such as now we support AQE on Databricks

Signed-off-by: Hao Zhu <9665750+viadea@users.noreply.github.com>

### CODE REVIEW

**Review by sameerz — CHANGES_REQUESTED** [review]
In docs/FAQ.md:

* Which distributions are supported? 
We should add CDS 3.3

* What hardware is supported? 
Should mention Pascal, as should the download page.  We test on P100.  

* How can I check if the RAPIDS Accelerator is installed and which version is running? 
We should update the version number from 22.06 to 22.10 in the example.  The full RAPIDS Accelerator and cudf build properties example are completely out of date and should be updated (can be done in a future release)

* What is the right hardware setup to run GPU accelerated Spark? 
This question is redundant from the "What hardware is supported?" two questions above and can be removed. 

* How does the performance compare to Databricks’ DeltaEngine? 
This question mentions that Delta is not open source.  Even with that, I am not sure it is valid to compare the performance of the plugin to a table layout format.  Can we modify this Q&A to say: 

```
Are table layout formats supported?
Yes, there is GPU support for Delta Lake read and Apache Iceberg 0.13 read.  Write operations will fall back to the CPU.  
```
We can link to docs/additional-functionality/iceberg-support.md

* Why is Avro library not found by RAPIDS? 
In this question, can we link 5796 to https://github.com/NVIDIA/spark-rapids/issues/5796 ?

File: docs/get-started/getting-started-databricks.md [hunk 1]
```diff
@@ -42,17 +32,17 @@ when using the plugin. Queries may still see significant speedups even with AQE
    
    See [issue-3143](https://github.com/NVIDIA/spark-rapids/issues/3143) for more detail.
 
-3. When selecting GPU nodes, Databricks requires the driver node to be a GPU node.  Outside of
+2. When selecting GPU nodes, Databricks requires the driver node to be a GPU node.  Outside of
```
Comment (sameerz) [inline docs/get-started/getting-started-databricks.md:35]: Do we want to comment that this limitation is with the Databricks UI, but the API interface allows selecting CPU driver nodes?

**Comment by viadea** [comment]
> In docs/FAQ.md:
> 
> * Which distributions are supported?
>   We should add CDS 3.3
> * What hardware is supported?
>   Should mention Pascal, as should the download page.  We test on P100.
> * How can I check if the RAPIDS Accelerator is installed and which version is running?
>   We should update the version number from 22.06 to 22.10 in the example.  The full RAPIDS Accelerator and cudf build properties example are completely out of date and should be updated (can be done in a future release)
> * What is the right hardware setup to run GPU accelerated Spark?
>   This question is redundant from the "What hardware is supported?" two questions above and can be removed.
> * How does the performance compare to Databricks’ DeltaEngine?
>   This question mentions that Delta is not open source.  Even with that, I am not sure it is valid to compare the performance of the plugin to a table layout format.  Can we modify this Q&A to say:
> 
> ```
> Are table layout formats supported?
> Yes, there is GPU support for Delta Lake read and Apache Iceberg 0.13 read.  Write operations will fall back to the CPU.  
> ```
> 
> We can link to docs/additional-functionality/iceberg-support.md
> 
> * Why is Avro library not found by RAPIDS?
>   In this question, can we link 5796 to #5796 ?

Done. Take a look again?

File: docs/FAQ.md [hunk 2]
```diff
@@ -364,11 +356,11 @@ Yes
 The GPU is not needed on the driver and there is no benefit to having one available on the driver
 for the RAPIDS plugin.
 
-### How does the performance compare to Databricks' DeltaEngine?
+### Are table layout formats supported??
```
Comment (sameerz) [inline docs/FAQ.md:359]:
```suggestion
### Are table layout formats supported?
```

**Review by sameerz — CHANGES_REQUESTED** [review]
One nit.

**Review by sameerz — APPROVED** [review]

**Comment by sameerz** [comment]
build

**Pull request merged**